mmv1/third_party/terraform/services/logging/resource_logging_bucket_config.go

```diff
@@ -287,6 +314,10 @@ func resourceLoggingBucketConfigRead(d *schema.ResourceData, meta interface{}) e
 		return fmt.Errorf("Error setting cmek_settings: %s", err)
 	}
 
+	if err := d.Set("index_configs", flattenIndexConfigs(res["indexConfigs"].([]interface{}))); err != nil {
```

Comment:
Ah, we're dying on this field when the value is `nil`. We'll want to change this to just `d.Set("index_configs", flattenIndexConfigs(res["indexConfigs"]));` and handle the value inside the flatten function. https://github.com/hashicorp/terraform-provider-google-beta/blob/35d78ff10768b60b27d020d08fde466d43826470/google-beta/services/compute/resource_compute_subnetwork.go#L1124-L1142 is a good reference.

Reply:
I see. Thanks for taking time with it, will get that updated now 👍